src/settings/tabtexteditor.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * class provides services:
+ *  changing editor' font
+ *  changing editor' font size
+ *  show selected font
+*/
+#ifndef TABTEXTEDITOR_H
+#define TABTEXTEDITOR_H
+
+#include <QWidget>
+#include <QtWidgets>
+#include <QHBoxLayout>
```

Comment:
Header QtWidgets includes all elements of Qt UI
It is huge
You must remove this include from your files, especially - from headers
**_It might cause identifiers conflict_**

Reply:
removed